addons/payment_adyen/controllers/main.py

```diff
@@ -120,6 +121,9 @@ def adyen_payments(
             'recurringProcessingModel': 'CardOnFile',  # Most susceptible to trigger a 3DS check
             'shopperIP': payment_utils.get_customer_ip_address(),
             'shopperInteraction': 'Ecommerce',
+            'shopperEmail': partner_sudo.email,
+            'shopperName': partner_sudo.name,
+            'telephoneNumber': partner_sudo.phone,
```

Comment:
The new parameters should also be added in `payment.transaction::_send_payment_request()` for payments by token.